subgraph/networks.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mainnet": {
+    "StreamFactory": {
+      "address": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
```

Comment:
@eladmallel is this address correct?

Reply:
this address is used for local testing; need to experiment further with subgraph local node config to land on a better network name

Reply:
ok, updated the local testing network name to `local`